experimental/torch_xla2/examples/torchbench_models/BERT_pytorch.py

```diff
@@ -0,0 +1,48 @@
+import torch
+import time
+import torch_xla2
+import torch_xla2.interop
+import os
+import importlib 
+import sys
+import logging
+import sys
+
+root = logging.getLogger()
+root.setLevel(logging.DEBUG)
+
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+handler.setFormatter(formatter)
+root.addHandler(handler)
+
+TORCH_BENCH_PATH = os.path.expanduser('~/git/qihqi/benchmark')
```

Comment:
Is this address intended?

Reply:
added a comment... without it doesnt work